### DISCUSSION TITLE
How to output page links from selected blog categories on a separate page?

### DISCUSSION THREAD

**Discussion by dragomano** [squidfunk/mkdocs-material#7055] [Q&A]
Let's say I created a `custom.md` page and I want to display an alphabetical index of all blog pages on it, but only from selected categories `cat1`, `cat2` (not all). How can I do that? Code examples?

```md
# Index

[Some link 1](cat1/some-slug)
[Some link 2](cat1/some-slug2)
[Some link 3](cat2/some-slug3)
```

**Comment by squidfunk** [comment]
You're describing what [tags](https://squidfunk.github.io/mkdocs-material/setup/setting-up-tags/) can do ☺️ Blog categories are currently not listable from anywhere, and we're currently considering it out of scope, given that the tags plugin supports this functionality, but we might reconsider if many users need it.

**Comment by dragomano** [comment]
Perhaps tags solve this problem, but only in the sponsored version that is unvavailable for me. I have no bank accounts abroad and my PayPal is blocked as I live in Russia.

There is a plugin [mkdocs-blogging-plugin](https://github.com/liang2kl/mkdocs-blogging-plugin), it has [functionality that seems to solve such a problem](https://liang2kl.github.io/mkdocs-blogging-plugin/#categories).

For example, to display a list of posts from selected categories on an index page, we can use this syntax:

```twig
{{ blog_content category1 }}
{{ blog_content category2 }}
{{ blog_content category5 }}
```

**Reply by squidfunk** [reply]
The [tags plugin](https://squidfunk.github.io/mkdocs-material/plugins/tags/) is part of the community edition and can be used without enrolling into Insiders. Yes, it got a major rewrite for Insiders, but what you're asking for is perfectly possible with the free version ☺️ Besides the tags plugin, there's also an open feature request that might go into the direction you're asking for, which is tracked in https://github.com/squidfunk/mkdocs-material/issues/5074. We can't currently put an estimate on when it will arrive, but we're definitely considering it. In the meantime, the tags plugin might be a good alternative.

Otherwise, if the blog plugin's current capabilities are not enough, you can also freely choose to use another blog plugin. There are [several to choose from](https://github.com/mkdocs/catalog?tab=readme-ov-file#-blogging), and Material for MkDocs ships only one possible, opinionated implementation 😉

**Reply by dragomano** [reply]
Ok, but let's say I already have a page with a list of tags in the menu. Can I add another tag page, but with a custom template that would only display links to posts from selected categories?

**Reply by squidfunk** [reply]
Multiple tags indexes and [configurable (and scoped) listings](https://squidfunk.github.io/mkdocs-material/setup/setting-up-tags/#advanced-features) are supported in Insiders, but currently not in the community edition. However, [once the next funding goal is reached](https://squidfunk.github.io/mkdocs-material/insiders/#16000-chipotle), this functionality will be available to everyone, which I'd expect to happen in the coming weeks/months ☺️
